composer.lock

```diff
@@ -846,7 +846,7 @@
                 "MIT"
             ],
             "description": "eLife Sciences API client",
-            "time": "2017-08-16T11:37:44+00:00"
+            "time": "2017-08-16 11:37:44"
```

Comment:
This needs to be reverted. (I think you also need to update your Composer as they changed date format a while back.)

Reply:
Ooops, sorry.